distributor-api-v1/pricing.md

```diff
@@ -0,0 +1,72 @@
+# Pricing
+
+* **Net price** is an amount without taxes.
+* **Gross price** is a net price + taxes.
```

Comment:
Here we're talking about price but there is nothing like that later on. There is only NetValue and GrossValue and that should already be described on the amount within operations. Also, within operations, I didn't find any link to amount. How is that possible if the API returns it?